app/main.py

```diff
@@ -1,3 +1,2 @@
-# TODO: add initial code
-def hello_world():
-    return "Hello, world!"
+def list_reverse(words: list):
```

Comment:
Wrong reference:
```
print(list_reverse(["Mate", "academy"]))  # ['ymedaca', 'etaM'] insteaf of ['ymed', 'acAetaM']
```

Reply:
@IvanRamyk It is ok, i had a wrong reference solution, now it fixed.